src/main/java/com/redhat/cloud/custompolicies/app/model/FullTrigger.java

```diff
@@ -65,14 +65,14 @@ private void storeActions(Trigger trigger, Policy policy) {
 
     String[] actionsIn = policy.actions.split(";");
     for (String actionIn : actionsIn) {
-      TriggerAction ta = new TriggerAction();
+      var ta = new TriggerAction();
       if (actionIn.trim().isEmpty()) {
         continue;
       }
       if (actionIn.toLowerCase().startsWith("email")) {
         ta.actionPlugin = "email";
       } else if (actionIn.toLowerCase().startsWith("webhook")) {
-        ta.actionPlugin = "webhook";
+        ta.actionPlugin = "hook"; // The legacy hooks apps
```

Comment:
It's hooks with ``s`` at the end ;)

https://github.com/RedHatInsights/custom-policies-engine/blob/master/external/src/main/java/com/redhat/cloud/custompolicies/engine/actions/plugins/BetaHooksActionPluginRegister.java#L26